.github/workflows/build.yml

```diff
@@ -27,6 +27,11 @@ jobs:
       - run: yarn run build
       - run: yarn run test
       - run: yarn run eslint
+      - run: yarn test:ci
+      - name: Generating coverage badges
+        uses: jpb06/jest-badges-action@latest
+          with:
+            branches: main
```

Comment:
A good question would be... do we want to generate a badge for `main` or the PR branch?